src/aggregate/aggregate_exec.c

```diff
@@ -132,6 +133,9 @@ static int sendChunk(AREQ *req, RedisModuleCtx *outctx, size_t limit) {
   nelem++;
   if (rc == RS_RESULT_OK && nrows++ < limit && !(req->reqflags & QEXEC_F_NOROWS)) {
     nelem += serializeResult(req, outctx, &r);
+  } else if (rc == RS_RESULT_ERROR) {
```

Comment:
This will cause errors to be sent inline, breaking the protocol. While I really like the idea, we cannot do this until 2.0

Reply:
Today 1.4 will return results, 1.6 is not, protocol already broken